tomviz/vtkCustomPiecewiseControlPointsItem.cxx

```diff
@@ -0,0 +1,52 @@
+/******************************************************************************
+
+  This source file is part of the tomviz project.
+
+  Copyright Kitware, Inc.
+
+  This source code is released under the New BSD License, (the "License").
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+******************************************************************************/
+#include "vtkCustomPiecewiseControlPointsItem.h"
+
+#include "vtkContextMouseEvent.h"
+#include "vtkContextScene.h"
+#include "vtkObjectFactory.h"
+
+//-----------------------------------------------------------------------------
+vtkStandardNewMacro(vtkCustomPiecewiseControlPointsItem);
```

Comment:
Tomviz, and if we move it there are more considerations than changing its style.